.gitignore

```diff
@@ -27,3 +27,5 @@
 /lib/seeds/*
 db/data.yml
 
+# Ignore environment variable file
+.env
```

Comment:
Is this intended to ignore `.env.dev` or just `.env`?

Reply:
I'm pretty sure I just copied the approach in cartographer: https://github.com/RockefellerArchiveCenter/cartographer